ejb-timer/README.adoc

```diff
@@ -35,6 +35,12 @@ include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
 include::../shared-doc/start-the-standalone-server.adoc[leveloffset=+1]
 // Build and Deploy the Quickstart
 include::../shared-doc/build-and-deploy-the-quickstart.adoc[leveloffset=+1]
+// Server Distribution Testing
```

Comment:
That new include wrt testing should be after accessing the app, before undeploy (which you already had).
PS: In this case leveloffset should be +1 to match the others

Reply:
fixed